### PR TITLE
fix: multi cozy token count bug

### LINKED ISSUE #2566: "Show All (x)" Feature in Multi Input Control
Dear Fundamentals Team,

we are using the component "Multi Input" quite intensively.
https://sap.github.io/fundamental-ngx/#/core/multi-input

But we haven't figured out yet what the additional feature of "Show All (x)" is supposed to do. We do not see any documentation. Playing around with the provided examples on the page mentioned above ends nowhere.

We kindly ask to point us to proper documentation of that feature or to briefly explain here what it is doing.

Many thanks,
Kai
image

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes #2566 

Addresses visual issues that arose due to changes in list component: see the bottom of the discussion on the linked issue

### CODE REVIEW

**Comment by mikerodonnell89** [comment]
> Clicking left ?
> clicking_left

This is the cozy token - for mobile devices - you don't have any arrow keys

File: libs/core/src/lib/list/list.component.scss [hunk 1]
```diff
@@ -376,16 +376,13 @@
 }
 .fd-list--dropdown .fd-list__item {
     height: auto;
-    padding: .75rem 1rem
+    padding: .75rem 1rem;
```
Comment (salarenko) [inline libs/core/src/lib/list/list.component.scss:379]: These style changes are an update from fundamental-styles?

Comment (mikerodonnell89) [reply libs/core/src/lib/list/list.component.scss:379]: > I don't think this is acceptable that MultiInput behaves differently based on cozy/compact mode. I think we should take care of it in separate ticket #2948.

@salarenko this is directly from the interaction specification, I've had a few conversations about it with the designers and this is what they want

**Comment by JKMarkowski** [comment]
could you change color of link element inside `fd-list__item:active` in this multi input to inherit? 
image

**Pull request merged**